app/views/categories/_form.html.erb

```diff
@@ -40,7 +40,8 @@
       Users must have at least this trust level to post in this category. Leave blank to allow
       anyone to post.
     </span>
-    <%= f.number_field :min_trust_level, class: 'form-element', min: 0, max: 10, step: 1 %>
+    <%= f.select :min_trust_level, options_for_select([['everyone', 0], ['anyone with a user account', 1], ['all but new users', 2], ['veteran users', 3], ['moderators only', 4], ['staff only', 5]], selected: @category.min_trust_level),
```

Comment:
Probably worth extracting these options out to a constant on the Ability model, so that we only ever have to change them in one place (L53 likewise).